conftest.py

```diff
@@ -30,3 +32,18 @@ def no_webpack_loader(monkeypatch):
         "webpack_loader.utils.get_files",
         lambda entry, extension=None, config='DEFAULT', attrs='': []
     )
+
+def pytest_json_modifyreport(json_report):
```

Comment:
Note that we have multiple `conftest.py` files for different sets of tests (and some of them import from each other):

```
./conftest.py
./common/test/conftest.py
./common/lib/conftest.py
./pavelib/paver_tests/conftest.py
./cms/conftest.py
```

Reply:
Is there any way to have this used everywhere besides modifying everyone of these files?